packages/hash/frontend/src/components/layout/PageHeader/SearchBar/index.tsx

```diff
@@ -155,34 +188,56 @@ const SearchBarWhenSearchIsEnabled: React.VFC = () => {
         },
         position: "relative",
         height: "100%",
-        ...(isMobile && displaySearchInput
-          ? {
-              position: "absolute",
-              width: "100%",
-              zIndex: 1,
-              left: 0,
-              top: theme.spacing(1.5),
-              px: 2,
-            }
-          : {}),
+        ...getSearchBarResponsiveStyles(isMobile, displaySearchInput),
```

Comment:
I think this is harder to read than just having:
```suggestion
        ...(isMobile && displaySearchInput
          ? {
              position: "absolute",
              width: "100%",
              zIndex: 1,
              left: 0,
              top: theme.spacing(1.5),
              px: 2,
            }
          : {}),
       mr: (isMobile && !displaySearchInput) ? 1 : undefined,
```
But this is fine for now.

Reply:
I would say no, the function has conditionals clearly defined. In the case of ternary operator, a lot of mental context has to be maintained to figure out which condition is applied where.